app/ui/cmd/Makefile

```diff
@@ -0,0 +1,25 @@
+ASSET_FILES=$(shell find ../dist/ -type f)
+
+# To enable debug:
+#BUILD_FLAGS=-gcflags "all=-N -l"
+
+all: linux
+
+../target/go/go-bindata:
+	go get github.com/go-bindata/go-bindata/go-bindata@v1.0.0
+	go build -o $@ github.com/go-bindata/go-bindata/go-bindata
+
+bindata_assetfs.go: ../target/go/go-bindata $(ASSET_FILES)
+	../target/go/go-bindata -prefix ../dist -o $@ ../dist/...
```

Comment:
Why you won't do put the static assets into the binary ? Would the standalone binary ever be used outside of a container, as its hard to adapt and to develop against it ?

As there is also data served from `/html` wouldn't it be easier and more straightforward to server everything from /html ?

Reply:
I could make that change easily.  Was thinking that by embedding them into the binary, might be less at risk of something not being put into the docker image properly during productization.

Reply:
Actually, I think it would be easier to avoid adding the assets within the binary for the following reasons:

* Less heap memory usage (not really sure how large the assets are)
* We could use a generic image which is built outside the UI build, freeing UI developers from running a go compiler each time
* Savin an additional pre-processor step saves some build times.

FYI for the operator, we also added the OpenShift template with gobin first, too but soon moved it out in the filesystem again. Also that the assets are more accessible in case some debug is required.